### PR TITLE
fix: adoption guide update

### PR DESCRIPTION
**Pull request by bdryanovski**
* updating getting started page
* adding adoption guide to the website.yml (Github Action)
* adding netlify configuration
* Moving eslint documentation into getting started at one single page
* Small ui fixes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by bbogdanov — APPROVED** [review]

File: netlify.toml [hunk 1]
```diff
@@ -280,6 +280,11 @@
   to = "/storybook/core"
   status = 200
 
+[[redirects]]
+  from = "/adoption-guide/*"
+  to = "/adoption-guide"
```
Comment (hippee-lee) [inline netlify.toml]: this needs to have a trailing slash

```to = "/adoption-guide/"```

You can also add a second redirect rule for `/adoption-guide` e.g:

```
[[redirects]]
  from = "/adoption-guide"
  to = "/adoption-guide/"
```

File: yarn.lock [hunk 2]
```diff
@@ -3329,7 +3329,7 @@
   integrity sha512-S9K+Q39BGOghyLHmR0Wdcmu1i1noSUk8HcvMj+3IaohZw02WFd99aPTQDHJeseXrXZP3CNovaSlePI0R11NcFg==
 
 "@cds/core@./dist/core":
-  version "5.0.0-rc.0"
+  version "5.0.0-rc.1"
```
Comment (hippee-lee) [inline yarn.lock:3332]: Should this and the other locked versions below be `5.0.1`?

**Review by hippee-lee — CHANGES_REQUESTED** [review]
I tried to access the app from the deploy preview but there seems to be an error. Doe sit work for you [here](https://6005b6c2ab6d727cd7aa7405--vmware-clarity.netlify.app/adoption-guide/)?

File: yarn.lock (hunk 2 above)
Comment (bdryanovski) [reply yarn.lock:3332]: Probably need just to merge with `next` .. it's an old PR - before the `v5` release.

File: packages/angular/projects/clarity-adoption/src/index.html [hunk 3]
```diff
@@ -10,4 +10,21 @@
   <body>
     <app-root></app-root>
   </body>
+
+  <!-- Hotjar Tracking Code for clarity.design/adoption-guide -->
+  <script>
```
Comment (hippee-lee) [inline packages/angular/projects/clarity-adoption/src/index.html:15]: I would rather not have the feedback form on anything except the website. We don't use it on storybook either. 
If we need analytics GA is a better option.

**Review by hippee-lee — COMMENTED** [review]
A few general thoughts:

1) since this is a standalone app it should live in `<root>/apps/`. I know we also build and publish the core storybook but that is heavily integrated with the core codebase. 
2) I'm not clear how folks will get to the application 
    - does it need something on the website so that people who want to see this example can find it?
3) will it be updated with more examples in the future? 

I'm sorry to keep blocking this ... I'm still unsure of the apps place as part of the website (and subdomains).

**Comment by gnomeontherun** [comment]
What is happening here?

**Comment by bdryanovski** [comment]
I Will try to update it on Monday, @hippee-lee is probably right that we need to see where we gonna locate the Adoption Guide and how we gonna promote it. Will try to draft some ideas and with pros and cons and will get back to it.

When it comes to the location, if it stays an Angular App I prefer to have it as part of the `packages/angular` so it will reuse the same angular version and tools that we use for the Angular and not have to maintain multiple versions. 

Gonna try to update or close it soon.

**Comment by hippee-lee** [comment]
We don't have to block this b/c of navigation to or promotion. 
I do think it should live in the apps directory though. The dev app there is a legacy holdover form the very early days and it _should_ eventually be deleted. As I mentioned, storybook is a different story but as a demonstration app this seems to be more similar to whats in the apps directory and I think that there is a script that syncs the build artifacts so that they do not need to be manually updated to use `@latest`. Shouldn't the adoption app use the current build of the angular libs?

**Comment by bdryanovski** [comment]
I gonna close this PR for now. 

Until then I will address the comments above.

**Pull request closed without merging**